pkg/networkmanager/interfaces.js

```diff
@@ -760,7 +760,7 @@ function NetworkManagerModel() {
             return;
 
         push_refresh();
-        cockpit.spawn(["udevadm", "info", obj.Udi]).
+        cockpit.spawn(["udevadm", "info", obj.Udi], { superuser: true }).
```

Comment:
If we classify accessing NM as a HACK, then we can remove this, I'd say.  Otherwise, we should probably run udev as the same user as we use to access NM.